tests/common.go

```diff
@@ -4912,7 +5002,7 @@ func ParseFlags() {
 	var torpedoJobType string
 	var anthosWsNodeIp string
 	var anthosInstPath string
-
+	log.Infof("The default scheduler is %v", defaultScheduler)
```

Comment:
This might have been missed during the debug message removal.

Reply:
Yes Sure. Will remove this along with other debug message which is not removed